contracts/eternal-storage/EternalStorage.sol

```diff
@@ -14,21 +14,25 @@ contract EternalStorage {
     // Address of the current implementation
     address internal _implementation;
 
+    // Address of the owner of the contract
+    address internal _owner;
+
+    /**
+     * @dev Access check: ensure that either we are on the genesis block, or
+     * `msg.sender` is the owner of the contract.  The genesis block is
+     * hard-coded into the client, so attacks based on it are not possible.
+     */
+    modifier onlyOwner() {
+        require(msg.sender == _owner || block.number == 0);
+        _;
+    }
+
     // Storage mappings
     mapping(bytes32 => uint256) internal uintStorage;
-    mapping(bytes32 => string) internal stringStorage;
```

Comment:
We shouldn't remove the storage mappings which are not used in the current code because this is eternal storage - we cannot know which mappings we need in the future, so we should leave all of them.

Reply:
@varasev I agree.